sdks/python/gen_protos.py

```diff
@@ -32,6 +29,7 @@
 import sys
 import time
 import warnings
+from importlib import import_module
```

Comment:
We probably no longer need to futurize generated code (see line 339). Assuming new version of proto generator produces a py3-compatible code. We could check the diff. 

This can be another change.

Reply:
I tried to generate_proto_files without futurize and found that it causes import error with urn files.
I'll investigate more.